criu/pie/restorer.c

```diff
@@ -221,6 +219,20 @@ static int restore_creds(struct thread_creds_args *args, int procfd, int lsm_typ
 		}
 	}
 
+	/*
+	 * Compare xids with current values. If all match then we can skip
+	 * setting them (which requires extra capabilities). There's no system
+	 * call to retrieve the fs ids so we'll just compare with the effective
```

Comment:
setfsuid returns the current fsuid. setfsuid((uid_t) -1) returns the current value without doing anything else.